draft-ietf-httpbis-connect-tcp.md

```diff
@@ -158,9 +159,9 @@ Clients SHOULD assume that all proxy resources generated by a single template sh
 
 In each HTTP version, any requirements related to closing connections in Classic HTTP CONNECT also apply to "connect-tcp", with the following modifications:
 
-* In HTTP/1.1, endpoints SHOULD close the connection in an error state to indicate receipt of a TCP connection error (e.g., a TCP RST or timeout).  Acceptable error states include sending an incomplete DATA capsule (as defined in {{Section 3.3 of !RFC9297}}), a TLS Error Alert ({{!RFC8446, Section 6.2}}), or a TCP RST (if TLS is not in use).  When a connection is terminated in an error state, the receiving endpoint SHOULD send a TCP RST if the underlying TCP implementation permits it.
-* In HTTP/2 and HTTP/3, senders MAY use an incomplete DATA capsule to indicate a TCP connection error, instead of (or in addition to) the signals defined for TCP connection errors in Classic HTTP CONNECT.  Recipients MUST recognize any incomplete capsule as a TCP connection error.
-* Intermediaries MUST propagate connection shutdown errors, including when translating between different HTTP versions.
+* If an endpoint in this specification receives a TCP FIN indicating clean shutdown, it MUST send a FINAL_DATA capsule.
+* When a connection is terminated without a FINAL_DATA capsule, the endpoint SHOULD send a TCP RST if the underlying TCP implementation permits it.
+* In HTTP/2 and HTTP/3, senders MAY additionally emit a stream error as in Classic HTTP CONNECT.
```

Comment:
Under what circumstances?

Reply:
I've removed this reference and instead listed the requirements explicitly.  The new explicit requirements are much more detailed, allowing us to cover @wtarreau's use case without additional capsules (... if all the RECOMMENDED behaviors are followed).